<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>516</width>
    <height>430</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QPushButton" name="button_RGB">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Pomiar</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioRed">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Red</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioBlue">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Blue</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioGreen">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Green</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioMagenta">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Magneta</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioYellow">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Yellow</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioCyan">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Cyan</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="button_R">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Ignored">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Red</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLCDNumber" name="lcdR">
            <property name="font">
             <font>
              <pointsize>36</pointsize>
             </font>
            </property>
            <property name="digitCount">
             <number>12</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QPushButton" name="button_G">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Ignored">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Green</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLCDNumber" name="lcdG">
            <property name="font">
             <font>
              <pointsize>36</pointsize>
             </font>
            </property>
            <property name="digitCount">
             <number>12</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QPushButton" name="button_B">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Ignored">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Blue</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLCDNumber" name="lcdB">
            <property name="font">
             <font>
              <pointsize>36</pointsize>
             </font>
            </property>
            <property name="digitCount">
             <number>12</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QPushButton" name="button_C">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Ignored">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Clear RGB</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLCDNumber" name="lcdC">
            <property name="digitCount">
             <number>12</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="0" column="1">
     <widget class="ColorWidget" name="widget" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>50</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <zorder></zorder>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>516</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="wlacz_led"/>
    <addaction name="wylacz_led"/>
    <addaction name="actionZapisz_pomiar"/>
    <addaction name="actionUstawienia_czujnika"/>
    <addaction name="actionZamknij"/>
   </widget>
   <widget class="QMenu" name="menuSkala">
    <property name="title">
     <string>Skala</string>
    </property>
    <addaction name="action10kHz"/>
    <addaction name="action100kHz"/>
    <addaction name="action600kHz"/>
   </widget>
   <addaction name="menuMenu"/>
   <addaction name="menuSkala"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="wlacz_led">
   <property name="text">
    <string>Włącz Led</string>
   </property>
  </action>
  <action name="wylacz_led">
   <property name="text">
    <string>Wyłącz Led</string>
   </property>
  </action>
  <action name="action10kHz">
   <property name="text">
    <string>10kHz</string>
   </property>
  </action>
  <action name="action100kHz">
   <property name="text">
    <string>100kHz</string>
   </property>
  </action>
  <action name="action600kHz">
   <property name="text">
    <string>600kHz</string>
   </property>
  </action>
  <action name="action_czerwony">
   <property name="text">
    <string>Czerwony</string>
   </property>
  </action>
  <action name="actionZapisz_pomiar">
   <property name="text">
    <string>Zapisz pomiar</string>
   </property>
  </action>
  <action name="actionZamknij">
   <property name="text">
    <string>Zamknij</string>
   </property>
  </action>
  <action name="actionUstawienia_czujnika">
   <property name="text">
    <string>Ustawienia czujnika</string>
   </property>
  </action>
  <action name="actionZr_b_pomiar">
   <property name="text">
    <string>Zrób pomiar </string>
   </property>
  </action>
  <action name="actionSprawd_R">
   <property name="text">
    <string>Sprawdź R</string>
   </property>
  </action>
  <action name="actionSprawd_G">
   <property name="text">
    <string>Sprawdź G</string>
   </property>
  </action>
  <action name="actionSprawd_B">
   <property name="text">
    <string>Sprawdź B</string>
   </property>
  </action>
  <action name="actionSprawd_C">
   <property name="text">
    <string>Sprawdź C</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>ColorWidget</class>
   <extends>QWidget</extends>
   <header>colorwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
